Bottom: 95443779b1e2b59c4bcb520c67594685d272287c
Top:    d6fc0eb1a4fe4f3464898c8bdaf66fe372367c2b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-21 21:29:44 +0000

Refresh of remove-hid-set_draw_xor-api

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 45b9b72..71754a5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -471,16 +471,6 @@ ghid_set_line_width (hidGC gc, Coord width)
   gc->width = width;
 }
 
-
-void
-ghid_set_draw_xor (hidGC gc, int xor)
-{
-  /* NOT IMPLEMENTED */
-
-  /* Only presently called when setting up a crosshair GC.
-   * We manage our own drawing model for that anyway. */
-}
-
 void
 ghid_set_draw_faded (hidGC gc, int faded)
 {
